ui/pages/settings/experimental-tab/experimental-tab.component.js

```diff
@@ -62,41 +59,77 @@ export default class ExperimentalTab extends PureComponent {
     } = this.props;
 
     return (
-      <div
-        ref={this.settingsRefs[1]}
-        className="settings-page__content-row--parent"
-      >
-        <div className="settings-page__content-item">
-          <span>{t('enableOpenSeaAPI')}</span>
-          <div className="settings-page__content-description">
-            {t('enableOpenSeaAPIDescription')}
+      <>
+        <div ref={this.settingsRefs[0]} className="settings-page__content-row">
+          <div className="settings-page__content-item">
+            <span>{t('enableOpenSeaAPI')}</span>
+            <div className="settings-page__content-description">
+              {t('enableOpenSeaAPIDescription')}
+            </div>
```

Comment:
Nit: Can we use more semantic tag than div here?

Reply:
Let's address this in a separate issue; unfortunately it's consistent with the rest of the `Settings` pages.